src/js/users/components/List.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * The MIT License (MIT)
+ * Copyright 2015 Government of Canada
+ *
+ * @author
+ * Ian Boyes
+ *
+ * @exports UsersList
+ */
```

Comment:
This is pretty useless. I think we should only have module comments if they have actual sentences in them.